Cargo.toml

```diff
@@ -6,7 +6,6 @@ members = [
     "packages/rs-drive",
     "packages/rs-platform-value",
     "packages/rs-drive-abci",
-    "packages/rs-drive-nodejs",
```

Comment:
We might want to keep this, after all bindings for node-js are a good thing to have, even if we don't use them in our own project.